### ISSUE TITLE
RuntimeError: CUDA error: the provided PTX was compiled with an unsupported toolchain.

### ISSUE THREAD

**Issue by Qualzz** [ashawkey/stable-dreamfusion#20]
Hello, I have this issue running the` python main.py --text "a hamburger" --workspace trial -O` command. 

I have the latest CUDA drivers and Cuda 11.8.1 installed. ( WSL 2 ) 

GPU: 3090

**Comment by ashawkey** [comment]
@Qualzz Hi, have you tried [this](https://forums.developer.nvidia.com/t/provided-ptx-was-compiled-with-an-unsupported-toolchain-error-using-cub/168292/2)?

**Comment by thuanz123** [comment]
best thing is to start with same pytorch and cuda from the README 
![image](https://user-images.githubusercontent.com/32274287/194683113-b97850a7-607d-47a2-bfe1-7f636ecd999f.png)

**Comment by Qualzz** [comment]
> @Qualzz Hi, have you tried [this](https://forums.developer.nvidia.com/t/provided-ptx-was-compiled-with-an-unsupported-toolchain-error-using-cub/168292/2)?

Yes, that's how I ended with the 11.8 version. 

Long story short despite having the correct pytorch/cuda combo, I had some conflict between my base environnement and the one I used for dreamfusion. It used the wrong version of cuda. 

Despite several reinstallation/cleaning, uninstallation of anaconda/conda, bunch of restarts, pytorch reinstall/cuda/nvidia drivers reinstall, I wasnt able to use it. 

I've spent a few hours trying to fix my installation, and out of nowhere, my last attempt of reinstalling pytorch ( with the exact same commands I used 10 times already ) now worked. 

So I can now close this issue.
